rpm/ngfd-plugin-droid-vibrator.spec

```diff
@@ -45,3 +45,4 @@ fi
 %defattr(-,root,root,-)
 %doc README COPYING
 %{_libdir}/ngf/libngfd_droid-vibrator.so
+%{_datadir}/ngfd/plugins.d/50-droid-vibrator.ini
```

Comment:
IIRC we should "own" the directories where rpm installs files by adding
%dir statements -> should have:

    %dir %{_libdir}/ngf
    %dir {_datadir}/ngfd/
    %dir {_datadir}/ngfd/plugins.d